Update README.md (#1)
cohandv/external-pvc-scaler: README.md

* This file can be mounted in `/app/config.ini` otherwise will take the default values in the `default.ini` file
* The only entity supported now is a PVC 

> ### Query DOES NOT GO WITH QUOTES! and consider that it travels on the query string, so store it with the appropiate format

Reviewer: ```suggestion
### Query DOES NOT GO WITH QUOTES! and consider that it is sent by query string, so store it with the appropiate format
```
